subprojects/solver/solver-smtlib/src/main/java/hu/bme/mit/theta/solver/smtlib/impl/cvc5/CVC5SmtLibItpSolver.java

```diff
@@ -115,6 +119,8 @@ public Interpolant getInterpolant(ItpPattern pattern) {
                     itpSolverBinary.issueCommand("(assert (= shevgcrjhsdfzgrjbms2dhrbcshdmrgcsh 0))");
                     itpSolverBinary.issueCommand(
                             String.format("(assert (and %s))", aTerm.collect(Collectors.joining(" "))));
+                    itpSolverBinary.issueCommand("(check-sat)");
```

Comment:
Is this needed or only here for debugging?

Reply:
Not needed, removed